templates/field/field--field-call-out-boxes.html.twig

```diff
@@ -0,0 +1,10 @@
+{% for row in items|batch(2) %}
+  <div{{ attributes.addClass('call-out-box row mx-1') }}>
+    {% for item in row %}
+     
+        {{ item.content }}
+ 
+    {% endfor %}
+  </div>
+
+{% endfor %}
```

Comment:
I don't think this file does anything. There is no longer a field called `field_call_out_boxes` as it was removed when moving to use the Layout Paragraphs module for layouts.